CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+![banner](assets/header-cover.png)
+
+Bienvenue et faites partie de notre communauté, en contribuant à quelque chose de significatif ! Votre contribution est très appréciée et nous attendons avec impatience l’opportunité de collaborer avec vous.
+
+Thank you | Merci 
+
+Il existe différentes manières de contribuer ,Vous pouvez ajouter :
+
+## Livres
+
+## Des articles
+
+## Discussions
+
+## Projets
+
+## Podcasts
+
+## Vidéos
+
+## Cours
+
+## Bourses
+
+## Certifications
+
+## Autre
+
+## Carte des contributions récentes
+![Alt](https://repobeats.axiom.co/api/embed/aa9ae67de6a5ef99ac1513cf90fd40ac2d16ebf8.svg "analytics image")
+
+## Mise en place du projet
+
+1. Compte GitHub
+
+    Créez si vous n'en avez pas, voir [ici](https://github.com/)
+
+2. Fork et Clone le Référentiel
+      ```
+      git clone https://github.com/[ your username GitHub ]/BLN-EduHub-Francophone
+      ```
+3. Ajoutez vos contributions
+
+4. Envoyez les modifications
+
+# English 
+
+Welcome and be a part of our community, contributing to something meaningful! Your input is highly valued, and we eagerly anticipate the opportunity to collaborate with you. 
+
+There is different ways you can contribute  , You can add  
+
+## Books
+
+## Articles
+
+## Talks
+
+## Projects
+
+## Podcasts
+
+## Videos
+
+## Courses
+
+## Fellowships
+
+## Certifications
+
+## Other
+
+## Recent Contributions Map
+![Alt](https://repobeats.axiom.co/api/embed/aa9ae67de6a5ef99ac1513cf90fd40ac2d16ebf8.svg "analytics image")
+
+1. Github Account 
+
+    Create if you do not have one see [here](https://github.com/)
```

Comment:
Hey @richarddushime , I don't think this section about `creating gh` accout is neccesary 😅